qemu/tests/virtio_port_hotplug.py

```diff
@@ -71,8 +72,12 @@ def run(test, params, env):
                 error_context.context("Load  module %s" % module, logging.info)
                 session.cmd("modprobe %s" % module)
         session.close()
+    logging.info("Kill the first serial process on host")
+    host_script = params['host_script']
+    os.system('pkill -f %s' % host_script)
```

Comment:
Please use `process`, and check the status.

Reply:
Thanks and update.